portal/patient/scripts/app/onsiteactivityviews.js

```diff
@@ -97,10 +97,26 @@ var actpage = {
 		        return eModal.ajax(params)
 		            .then(function () { });
 		 };
-
-		 $(document.body).on('hidden.bs.modal', function (){
-			 	window.location.href = './onsiteactivityviews';
-			});
+		// @todo below for emodal refactor
+        /*function showProfileModal(cpid) {
+            var title = 'Demographics Legend Red: Charted Values. Blue: Patient Edits Provider Audit.';
+
+            var params = {
+                buttons: [
+                    { text: 'Help', close: false, style: 'info btn-sm',id: 'formHelp'},
+                    { text: 'Cancel', close: true, style: 'default btn-sm'},
+                    { text: 'Revert Edits', close: false, style: 'success btn-sm',id:'replaceAllButton'},
+                    { text: 'Commit to Chart', style: 'danger btn-sm', close: false,id:'savePatientButton'}],
+                //onClosed: 'reload',
+                type: 'GET',
+                url: './patientdata?pid='+cpid+'&user='+cuser
+            };
+            dlgopen('','','modal-xl', 500, '', title, params);
+        }*/
```

Comment:
Is this commented out new code?

Reply:
for future when I get around to getting rid of emodal. though I could do while in that code but had issue so left to pick up later..